fem/hyperbolic.cpp

```diff
@@ -33,15 +56,15 @@ void HyperbolicFormIntegrator::AssembleElementVector(const FiniteElement &el,
    // shape function value at an integration point
    Vector shape(dof);
    // derivative of shape function at an integration point
-   DenseMatrix dshape(dof, el.GetDim());
+   DenseMatrix dshape(dof, Tr.GetSpaceDim());
```

Comment:
I don't believe that we can generalize this class to a general manifold without significant modifications. In that case, is it necessary to use `GetSpaceDim()` instead of `el.GetDim()`? It certainly improves readability, but this might give the impression that it works for general manifold as we distinguish `dim` and `sdim`.

Reply:
It is more a psychological question, I think 🙂 . I just did not want to keep around some obvious mistakes. Impression or not, the code will crash, but not here but at the point where the implementation is deficient for manifolds, i.e., multiplication of the flux and gradient. Anyway, not very important, I may change it back if you insist.